docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md

```diff
@@ -147,61 +155,345 @@ This is the disk volume of a node.
 
 For mount paths, use a mount point with enough space to contain your node density. Use `/data`. If you have multiple drives, add these as a comma-separated list: `/mnt/d0,/mnt/d1`
 
-
 ### Region and Zones
 
 Follow the steps below to provide the location of DB nodes. All these fields are user-defined, which will be later used during the universe creation.
 
-
 ![Configure On-Premises Cloud Provider](/images/ee/onprem/configure-onprem-3.png)
 
 ## Step 2. Provision the YugabyteDB nodes
 
 After finishing the cloud provider configuration, click on “Manage Instances” to provision as many nodes as your application requires:
 
-
 ### Manage Instances
 
 ![Configure On-Premises Cloud Provider](/images/ee/onprem/configure-onprem-4.png)
 
-1. Click on “Add Instances” to add the YugabyteDB node. You can use DNS names or IP addresses when adding instances.
-2. Instance ID is an optional user-defined identifier.
+1. Click Add Instances to add the YugabyteDB node. You can use DNS names or IP addresses when adding instances.
+1. Instance ID is an optional user-defined identifier.
 
 ![Configure On-Premises Cloud Provider](/images/ee/onprem/configure-onprem-5.png)
 
-### Run the pre-provisioning script
+### Provision the nodes manually
+
+To provision the nodes manually, you have two options:
+
+* If the SSH user you provided has sudo privileges but requires a password, follow the steps in [Run the pre-provisioning script](#run-the-pre-provisioning-script).
+
+* If the SSH user does not have sudo privileges, follow the steps in [Set up database nodes manually](#set-up-database-nodes-manually).
+
+#### Run the pre-provisioning script
 
 {{< note title="Note" >}}
-This step is only required if you set “Manually Provision Nodes” to true otherwise, you need to skip this step.
+This step is only required if you set “Manually Provision Nodes” to true; otherwise, you need to skip this step.
```

Comment:
Since we've two ways of manually provisioning, can you change this to:
_This step is only required if you set “Manually Provision Nodes” to true AND the SSH user has sudo privileges which requires password; otherwise, you need to skip this step._